lib/filesystem.js

```diff
@@ -139,6 +139,10 @@ class Filesystem {
     followLinks = typeof followLinks === 'undefined' ? true : followLinks
     const info = this.getNode(p)
 
+    if (typeof info === 'undefined') {
+      return undefined
```

Comment:
Why not throw an error here?

Reply:
Fixed in c555f9cf73dad857f9f2974d8cadcb6ecafb5745.